### ISSUE TITLE
Set Working Directory should allow for imports of local files

### ISSUE THREAD

**Issue by samholmes** [lukehaas/runjs#10]
Set Working Directory currently only allows me to import modules from the directory's node_module directory. It doesn't allow me to import modules that are local such as `require('./lib/api.js')` for example. Support for this would be useful for quick testing for your project's codebase.

**Comment by lukehaas** [comment]
Good point. I'll aim to implement this in the next release.

**Comment by lukehaas** [comment]
This is now implemented in version 1.3.0

**Comment by jguffey** [comment]
I think this issue is back in 1.5.0. I can't import local files, nor can I import `node_modules`. 

I tried setting the working directory to my site root, and also to node_modules itself and both seem to fail right now.

Also tried:
```JS
import moment from 'moment';
console.log(moment); --> Error: Cannot find module 'moment'
```

![Screen Shot 2019-05-31 at 1 28 15 PM](https://user-images.githubusercontent.com/238058/58733016-2204e000-83a8-11e9-8a6f-bb7b32d1ef68.png)

**Comment by lukehaas** [comment]
@jguffey This is a new issue, unrelated to this original enhancement ticket.
I've created a new issue to track it: https://github.com/lukehaas/RunJS/issues/78
I'm aiming to release a solution very soon.

**Comment by lukehaas** [comment]
@jguffey This is now fixed in 1.5.1. Thanks for catching it.

**Comment by jguffey** [comment]
No problem, thanks for addressing it so quickly!
